templates/index.html

```diff
@@ -144,7 +144,9 @@ <h1>Article Classification</h1>
 	<script type="text/javascript">
 		$(document).ready(function() {
 			$('form label.placeholder').each(function() {
-				$(this).addClass('off');
```

Comment:
This change guarantees that if the fields on the form are pre-filled, the text label is above the form field instead of overlapping.